pawn/tf2items.inc

```diff
@@ -201,17 +201,17 @@ native Float:TF2Items_GetAttributeValue(Handle:hItem, iSlotIndex);
  *
  * @param client				Client Index.
  * @param classname				The classname of the entity that will be generated.
- * @param iItemDefinitionIndex	Item definition index.
+ * @param weapon	Item definition index.
  * @param hItem					Handle to a TF2Item object wich describes what values will be overriden.
  */
-forward Action:TF2Items_OnGiveNamedItem(client, String:classname[], iItemDefinitionIndex, &Handle:hItem);
+forward Action TF2Items_OnGiveNamedItem(int client, char[] classname, int weapon, Handle &hItem);
 
-forward TF2Items_OnGiveNamedItem_Post(client, String:classname[], itemDefinitionIndex, itemLevel, itemQuality, entityIndex);
+forward void TF2Items_OnGiveNamedItem_Post(int client, char[] classname, int weapon, int level, int quality, int entity);

```

Comment:
This is sadly a breaking change, should probably make it `int` to preserve compatibility.